src/runtimeConfig.json

```diff
@@ -0,0 +1,3 @@
+{
+    "minimumEngineVersion": "0.0.0"
```

Comment:
This shouldn't be necessary... Just use the package.json version (CLI & Engine versions should be aligned)